policy/templates/terraform/gpac/tyk.auto.tfvars

```diff
@@ -0,0 +1,13 @@
+tyk_release_branches = [
+{{- with $repo := index .RepoPolicies "tyk" }}
+{{- range $branch, $values := $repo.ReleaseBranches }}
+{ branch    = "{{ $branch }}",
+	reviewers = "{{ $values.ReviewCount }}",
+	convos    = "{{ $values.Convos }}",
+	{{- if $values.SourceBranch }}
+	source_branch  = "{{ $values.SourceBranch }}",
+	{{- end }}
+	required_tests = ["{{ $values.Tests | join "," }}"] },
```

Comment:
@alephnull , @asutosh , line 10 is something I'm still struggling with, I need to generate a comma separated list of quoted strings.
I have tried this option using the quote function, but is printing some weird characters when rendering the templates

```suggestion
	required_tests = ["{{ $values.Tests | quote | join "," }}"] },
```

<img width="865" alt="image" src="https://user-images.githubusercontent.com/35462288/205154391-735fc8bd-2515-4f23-8a47-d45693217d36.png">